app/Controllers/CtrlAdminEmail.php

```diff
@@ -24,7 +24,16 @@ public function viewSpecificEmails($id)
 
     public function deleteEmail()
     {
-        $isDeleted = true;
+        $POST       = $this->request->getPost();
+        $emailId    = $POST['emailId'];
```

Comment:
```suggestion
        $emailId       = $this->request->getPost('emailId');
```
You can get the email id directly from the post and it would be good if you use a try catch for database exception handling